src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
```

Comment:
You've touched the line, it's new code now : )
Leave the campsite cleaner than you found it. Sorry if this is annoying, but we can't punt every clean up for another day that never comes.